website/docs/d/search_analyzer.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search analyzers"
+sidebar_current: "docs-mongodbatlas-resource-search-analyzers"
+description: |-
+Provides a Search Analyzers resource.
+---
+
+# mongodbatlas_search_analyzer
+
+`mongodbatlas_search_analyzer` allows you to retrieve and edit Atlas Search [analyzers](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-custom-analyzers) and [index configurations](https://docs.atlas.mongodb.com/atlas-search/#std-label-fts-top-ref) for the specified cluster.
+
+## Example Usage
+
+```hcl
+data "mongodbatlas_search_analyzer" "test" {
+  project_id = "<PROJECT_ID>"
+  cluster_name = "<CLUSTER_NAME>"
+  page_num = 1
+  items_per_page = 100
```

Comment:
We don't have pagination number like this in any other data source so I'm curious why we are doing it here? (figured it was easier to comment on the example than the data sources).

Reply:
indeed, in mongodbatlas_projects also uses pagination
e.g.

``` data "mongodbatlas_project" "test" {
  page_num = 1
  items_per_page = 5
}
```

but I you think that this is not necessary, I can remove it

what do you think?

Reply:
That is a good point, I forgot it was in projects! But it isn't there for other similar data sources which makes me think we are missing them (I'll create a ticket to investigate later).  Let's leave it for now then.  Feel free to resolve this conversation after you read this.